dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ErrorCode.java

```diff
@@ -247,7 +247,12 @@
     E7301( "At least one organisation unit group set must be specified" ),
 
     /* Debug analytics */
-    E7400( "Debug query must contain at least one data element, one period and one organisation unit" );
+    E7400( "Debug query must contain at least one data element, one period and one organisation unit" ),
+
+    /* Validation Results API */
+    E7500( "Query filter for organisation unit does not exist: `{0}`" ),
```

Comment:
Change to just "Organisation unit does not exist" for consistency. Same for validation rule and period.